docs/content/configuration/include/_index.md

```diff
@@ -70,4 +70,117 @@ global {
 Included configuration files may use any supported format and settings are merged so that multiple files can extend the same profiles.
 The HCL format is special in that it cannot be mixed with other formats.
 
-Included files cannot include nested files and do not change the current configuration path.
+Included files cannot include nested files. Specifying `includes` inside an included file has no effect.
+
+Within included files, the current [configuration path](../#path-resolution-in-configuration) is not changed. Path resolution remains relative to the path of the main configuration file.
+
+## Configuration Merging
+
+Configuration files are loaded and applied in a fixed order:
+
+1. The main configuration file is loaded first
+2. `includes` are iterated in declaration order:
+   * Every item may be a single file path or glob expression
+   * Glob expressions are resolved and iterated in alphabetical order
+   * All paths are resolved relative to [configuration path](../#path-resolution-in-configuration)
+
+Configuration files are loaded in the following order when assuming `/etc/resticprofile/profiles.conf` with `includes = ["first.conf", "conf.d/*.conf", "last.conf"]`:
+```
+/etc/resticprofile/profiles.conf
+/etc/resticprofile/first.conf
+/etc/resticprofile/conf.d/00_a.conf
+/etc/resticprofile/conf.d/01_a.conf
+/etc/resticprofile/conf.d/01_b.conf
+/etc/resticprofile/last.conf
+```
+
+Configuration **merging** follows the logic:
+
+* Configuration properties are replaced
+* Configuration structure (tree) is merged
+* What includes later overrides what defines earlier
+* Lists of values or objects (tree structure) are considered properties not config structure and will be replaced
```

Comment:
as such it's not part of the "merging" process, but should we also note where the go template gets resolved in the grand scheme of things?

Reply:
Makes perfect sense to me. Added it as introduction to "loading" as it's important that this is understood.